samples/README.md

```diff
@@ -0,0 +1,77 @@
+# Google Cloud Tasks Pull Queue Samples
+
+[![Open in Cloud Shell][shell_img]][shell_link]
+
+[shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
+[shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-tasks&page=editor&open_in_editor=samples/README.md
+
+Sample command-line program for interacting with the Google Cloud Tasks API
+using pull queues.
+
+Pull queues let you add tasks to a queue, then programatically remove and
+interact with them. Tasks can be added or processed in any environment,
+such as on Google App Engine or Google Compute Engine.
+
+`tasks.js` is a simple command-line program to demonstrate listing queues,
+ creating tasks, and pulling and acknowledging tasks.
```

Comment:
what does "acknowledging" a task mean?

Reply:
The worker must acknowledge the task after they have performed the work associated with the task. Should the readme include this information?

Reply:
That's what I was getting at, but on second thought I'm not sure what the documentation context for this might be.